test_requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/OCA/account-invoicing@refs/pull/1050/head#subdirectory=setup/account_invoice_section_sale_order
```

Comment:
this has to be dropped. Anyway. The file name is wrong: should test-requirements.txt :wink: